src/Message/PurchaseResponse.php

```diff
@@ -2,16 +2,10 @@
 
 namespace Ampeco\OmnipayPayMob\Message;
 
-use Ampeco\OmnipayPayMob\Gateway;
-
 class PurchaseResponse extends Response
 {
-    public function isSuccessful()
+    public function isPending()
     {
-        if ($this->data['is_auth'] === 'true') {
-            return $this->statusCode < 400;
-        }
-
-        return parent::isSuccessful() && $this->getCode() === Gateway::STATUS_SUCCESS;
+        return $this->data['pending'];
```

Comment:
From what I see in $this->data['is_auth'] === 'true' looks like pending can be true/false as string, please double check it that's true, because if it is string $this->data['pending'] will always be true